### DISCUSSION TITLE
Forcesay displays dialogue twice in speech bubbles in multiplayer

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15395] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

![20241218163401](https://github.com/user-attachments/assets/919fcb35-d3ec-4c65-ae59-b53dbe820e90)
one in client language
one in server language

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by TheNinjaScout** [comment]
Potential fix: https://github.com/FakeFishGames/Barotrauma/pull/15400

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/0e7e4b702adba5b98936424ddb2bd8936543fa9e, fix coming up in the next unstable build.
